packages/utilities/src/utilities/sameheight/sameheight.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * Utility that sets elements under a same parent to the sameheight.
+ *
+ * It uses the selector passed to the `sameheight` attribute, to identify
+ * which items should be aligned
+ *
+ * @example
+ * `HTML`
+ *    <div sameheight=".adjustme">
+ *      <elemA class="adjustme">
+ *      <elemB class="adjustme">
+ *    </div>
+ *
+ * `JS`
+ * import {sameheight} from '@carbon/ibmdotcom-utilities';
+ *
+ * sameheight();
+ */
+function sameheight() {
+  /**
+   * Internal function made for avoiding adding more eventlisteners
+   */
+  function setHeight() {
+    if (window.innerWidth > 671) {
+      const sameheightContainers = Array.prototype.slice.call(
+        document.querySelectorAll('[sameheight]')
```

Comment:
It also looks like this is going to need to be executed at a global level. I wonder if it would make more sense to allow a class name to be passed into the utility so this could be used at the component level. That way we could better leverage react hooks and not have to worry as much about racing issues.

Reply:
I actually made the suggestion in the channel to pass in the array of elements instead, that way a lot of this classname/queryselectorall logic is taken out of the utility.

Reply:
I've done this in the last commit